tests/test_plugin.py

```diff
@@ -239,6 +239,37 @@ def test_register_host():
 
 
 @with_setup(lib.setup_empty, lib.teardown)
+def test_current_target():
+    """pyblish.api.current_target works"""
+    pyblish.plugin.register_target("mytarget")
+    assert_equals(pyblish.plugin.current_target(), "mytarget")
+
+    assert_raises(Exception, pyblish.plugin.deregister_target, "notExist")
+
+
+@with_setup(lib.setup_empty, lib.teardown)
+def test_current_target_latest():
+    """pyblish.api.current_target works"""
+    pyblish.plugin.deregister_all_targets()
+    pyblish.plugin.register_target("mytarge1")
```

Comment:
"targe1" should be "target1", have a look at surrounding similar misspellings as well.